src/pykeen/nn/functional.py

```diff
@@ -611,7 +611,7 @@ def rotate_interaction(
         The scores.
     """
     # r expresses a rotation in complex plane.
-    h, r, t = [view_complex(x) for x in (h, r, t)]
```

Comment:
are we ready to replace this function with the torch builtin?

Reply:
except in testing and dead code, we do not seem to reference this function anywhere else